src/sentry/api/endpoints/project_transaction_names.py

```diff
@@ -38,24 +38,12 @@ def get(self, request: Request, project) -> Response:
         limit = int(params.get("limit", 1000))
         merge_threshold = int(params.get("threshold", 100))
         return_all_names = params.get("returnAllNames")
-        namespace = params.get("namespace")
-
-        if namespace == "spans":
-            namespace = ClustererNamespace.SPANS
```

Comment:
`ClustererNamespace.SPANS` does not exist. https://github.com/getsentry/sentry/blob/65dd81feee1777cc40c0f3148a24989f6d0f74c6/src/sentry/ingest/transaction_clusterer/__init__.py#L23